recipe/build-one.sh

```diff
@@ -15,12 +16,27 @@ if [[ "${target_platform}" != "${build_platform}" ]]; then
   export CMAKE_ARGS="${CMAKE_ARGS} -DCMAKE_CROSSCOMPILING=ON"
 fi
 
-# can add lagraph and -DSUITESPARSE_USE_SYSTEM_GRAPHBLAS after packaging 9.1
+if [[ -z "${FC:-}" || ! -f "${FC:-fc_compiler}" ]]; then
+  echo "Not using fortran"
+  export CMAKE_ARGS="${CMAKE_ARGS} -DSUITESPARSE_USE_FORTRAN=OFF"
+fi
+
+cd "${SUBPKG_DIR}"
+
 cmake -B build \
-  -DSUITESPARSE_ENABLE_PROJECTS="suitesparse_config;amd;btf;camd;ccolamd;colamd;cholmod;cxsparse;ldl;klu;umfpack;paru;rbio;spqr;spex" \
+  -G Ninja \
   -DBLA_VENDOR="Generic" \
   -DBUILD_SHARED_LIBS=ON \
   -DBUILD_STATIC_LIBS=OFF \
   ${CMAKE_ARGS}
-cmake --build build --parallel "${CPU_COUNT:-1}" --verbose
+cmake --build build --parallel "${CPU_COUNT:-1}"
 cmake --install build --verbose
+
+# verify subpackage versions
+if [[ "${PKG_NAME}" == lib* ]]; then
+  if [[ "$target_platform" == osx-* ]]; then
+    test -f ${PREFIX}/lib/${PKG_NAME}.${PKG_VERSION}${SHLIB_EXT}
+  else
+    test -f ${PREFIX}/lib/${PKG_NAME}${SHLIB_EXT}.${PKG_VERSION}
+  fi
+fi
```

Comment:
OK, just saw that the subpackage versions are validated here (somewhat implicitly) using the SOVERSIONS. The problem is that this won't trigger if upstream bumps the version but doesn't change the SOVERSION (not sure how realistic this is; it may be good enough as-is).

Reply:
I updated this to check the CMAKE_PROJECT_VERSION directly instead of SOVERSION